libs/accounts/src/lib/transfer-form.tsx

```diff
@@ -193,21 +193,17 @@ export const TransferForm = ({
               onValueChange={(value) => {
                 field.onChange(value);
               }}
-              placeholder={t('Please select')}
+              placeholder={t('Please select an asset')}
               value={field.value}
             >
               {assets.map((a) => (
-                <Option key={a.id} value={a.id}>
-                  <div className="text-left" data-testid={`asset-${a.id}`}>
-                    <div>{a.name}</div>
-                    <div className="text-xs">
-                      <span className="font-mono" data-testid="asset-balance">
-                        {formatNumber(a.balance, a.decimals)}
-                      </span>{' '}
-                      <span>{a.symbol}</span>
-                    </div>
-                  </div>
-                </Option>
+                <AssetOption
+                  key={a.id}
+                  asset={a}
+                  balance={
+                    <Balance balance={a.balance.toString()} symbol={a.symbol} />
```

Comment:
formatNumber and new BigNumber().toString() will give different results.  We should always use formatNumber